src/services/redisSessionStorage.ts

```diff
@@ -24,9 +28,9 @@ export default class RedisSessionStorage implements ISessionStorage {
    * {@inheritDoc}
    */
   public async set(user: User): Promise<Either<Error, boolean>> {
+    // Set key to hold the string value. If key already holds a value, it is overwritten, regardless of its type.
```

Comment:
unstage this change as it's not in the scope of this PR

Reply:
fixed on 2354a0b